protostar/commands/test/test_cases.py

```diff
@@ -20,13 +21,54 @@ def get_formatted_file_path(self):
 @dataclass(frozen=True)
 class PassedTestCase(TestCaseResult):
     test_case_name: str
-    tx_info: Optional[StarknetTransactionExecutionInfo]
+    execution_resources: Optional[ExecutionResourcesFacade]
 
     def __str__(self) -> str:
-        result: List[str] = []
-        result.append(f"[{log_color_provider.colorize('GREEN', 'PASS')}]")
-        result.append(f"{self.get_formatted_file_path()} {self.test_case_name}")
-        return " ".join(result)
+        first_line_elements: List[str] = []
+        first_line_elements.append(f"[{log_color_provider.colorize('GREEN', 'PASS')}]")
+        first_line_elements.append(
+            f"{self.get_formatted_file_path()} {self.test_case_name}"
+        )
+
+        if self.execution_resources:
```

Comment:
Where does `self.execution_resources` come from, and what is the difference between it and the `execution_resources` argument?

Reply:
`self.execution_resources` refers to `execution_resources` from the 24. There's no difference between them.

https://docs.python.org/3/library/dataclasses.html

Reply:
🤦🏻 misunderstood the code